<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XVideoUiClass</class>
 <widget class="QWidget" name="XVideoUiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XVideoUi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#XVideoUiClass
{
	
	background-color: rgb(53, 53, 53);
}
#pushButtonClose
{
	font: 75 11pt &quot;等线&quot;;
	color:rgb(255,255,255);
    background-color: rgb(53, 53, 53);
}

QPushButton:hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
    border: none;
    border-radius:5px;
    color: rgb(255, 255, 255);
    font: 75 12pt &quot;黑体&quot;;
 }
QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
    border: none;
    border-radius:5px;
    color: rgb(255, 255, 255);
    font: 75 12pt &quot;黑体&quot;;
 }

QLabel
{
	border-radius:5px;
    color: rgb(255, 255, 255);
    font: 75 12pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="XVideoWidget" name="src1Video">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>400</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="XVideoWidget" name="src2Video">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>400</width>
     <height>320</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOpenFile">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>10</y>
     <width>901</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>818</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonClose">
      <property name="minimumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSlider" name="SliderMovie">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBright">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="SpinBoxContrast">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <double>1.000000000000000</double>
   </property>
   <property name="maximum">
    <double>3.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSet">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExport">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>390</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPlay">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRotate">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxFlip">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>520</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>不处理</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>左右</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>上下</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>旋转</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxX">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>560</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxY">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>580</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pyUp">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>630</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pyDown">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>610</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="cX">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>670</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cY">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>670</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cW">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>670</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="cH">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>670</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxColor">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>490</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>RGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>GRAY</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonMark">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>水印</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="MarkX">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>480</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="MarkY">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>480</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999</number>
   </property>
  </widget>
  <widget class="XVideoWidget" name="src3Video">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>400</width>
     <height>280</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBlend">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>480</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>融合</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="BlendA">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>480</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="MarkA">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>480</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>99999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMerge">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>460</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>合并</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>430</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>亮度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>对比度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>463</x>
     <y>490</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转角度</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>490</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>灰显</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>520</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>镜像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>560</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>裁剪X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>580</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>裁剪Y</string>
   </property>
  </widget>
  <widget class="QSlider" name="SliderMovie_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>430</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="SliderMovie_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>XVideoWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>XVideoWidget.h</header>
   <slots>
    <slot>SliderPress()</slot>
    <slot>SliderRealse()</slot>
    <slot>SliderMove(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XVideoUi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>SliderMovie</sender>
   <signal>sliderPressed()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>SliderPress()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderMovie</sender>
   <signal>sliderReleased()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>SliderRelease()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderMovie</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>SliderMove(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSet</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonExport</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Export()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>567</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPlay</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Play()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonPause</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Pause()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>-4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMark</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Mark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBlend</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Blend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonMerge</sender>
   <signal>clicked()</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Merge()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderMovie_3</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Left(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>-18</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SliderMovie_2</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>XVideoUiClass</receiver>
   <slot>Right(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>439</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SliderPress()</slot>
  <slot>SliderRelease()</slot>
  <slot>SliderMove(int)</slot>
  <slot>Set()</slot>
  <slot>Export()</slot>
  <slot>Play()</slot>
  <slot>Pause()</slot>
  <slot>Mark()</slot>
  <slot>Blend()</slot>
  <slot>Merge()</slot>
  <slot>Left(int)</slot>
  <slot>Right(int)</slot>
 </slots>
</ui>
